### ISSUE TITLE
Misleading sources of documentation

### ISSUE THREAD

**Issue by jamesp** [SciTools/iris#4286]
## 📚 Documentation

Where is this documentation being produced? Can we turn it off? It is out of date.

https://scitools-docs.github.io/iris/master/index.html

**Comment by tkknight** [comment]
Is it now understood where this doc instance comes from?

**Comment by rcomer** [comment]
Is this the culprit? https://github.com/SciTools-docs/iris

**Comment by rcomer** [comment]
#2610 👀

**Comment by jamesp** [comment]
Yep that looks like it might be it!

**Comment by jamesp** [comment]
It looks like it previously was being published by Jenkins, but no longer is.  I don't have access to the control panel for this account.  Could someone else in @SciTools/admins disable it, or redirect (maybe http 301 if possible) to our RTD instance?

**Comment by pp-mo** [comment]
> Could someone else in @SciTools/admins disable it

Agreed, it seems with the permission, you can just disable the github "Pages" option
https://guides.github.com/features/pages/#setup

**Comment by tkknight** [comment]
I do not seem to have permissions to do anything on https://github.com/SciTools-docs/iris.  

For reference, we have https://github.com/SciTools/scitools.org.uk/tree/master/iris/docs which is the docs for **Scitools** that includes links to the docs for **Iris** and **Cartopy** published via https://scitools.org.uk/.  After Iris 2.4 we moved to ReadTheDocs (RTD) and will auto redirect to RTD when selecting the latest Iris.

If someone has permissions for https://github.com/SciTools-docs/iris, I think we can either disable the pages or add a simple redirect like we do on the scitools.org.uk repo, see https://github.com/SciTools/scitools.org.uk/blob/master/iris/docs/stable_readthedocs/index.html - as per @jamesp suggested.

Perhaps @bjlittle has permissions, or @pelson who I note was the first to [commit ](https://github.com/SciTools-docs/iris/commits/master)to master before the automated travis commits.
